src/main/java/org/sagebionetworks/web/client/widget/entity/browse/FilesBrowser.java

```diff
@@ -62,7 +69,13 @@ public void onProgrammaticDownloadOptions() {
 
 	@Override
 	public void onAddToDownloadList() {
```

Comment:
Adding all file children to the DL from a container (folder/project). Use new SRC DownloadConfirmation component (wrapped in AddToDownloadListV2 in SWC) if in alpha mode.